capstone/src/components/User/Adventurer/Preference/FilterMenu/FilterMenu.jsx

```diff
@@ -0,0 +1,37 @@
+import { ChevronDownIcon } from '@chakra-ui/icons';
+import {
+  Menu, MenuButton, Button, MenuList, MenuItem,
+} from '@chakra-ui/react';
+
+// inactivePreferences={inactivePreferences}
+//           setInactivePreferences={setInactivePreferences}
+//           setActivePreferencesIDs={setActivePreferencesIDs}
+function handleAddition(objectId, inactivePreferences, setInactivePreferences, activePreferencesIDs, setActivePreferencesIDs) {
+  setActivePreferencesIDs([...activePreferencesIDs, objectId]);
+  setInactivePreferences(
+    inactivePreferences.filter((preferences) => preferences.objectId !== objectId),
+  );
+}
+
+export default function FilterMenu({
+  inactivePreferences, setInactivePreferences, activePreferencesIDs, setActivePreferencesIDs,
+}) {
+  return (
+    <Menu>
+      <MenuButton as={Button} rightIcon={<ChevronDownIcon />} width="1000px">
```

Comment:
I'd be cautious of hard-coding widths, especially large ones like this, because it can break your layout at different screen sizes.

Previously I saw you used css grid, which is a great alternative. Or even better, chakra has layout capabilities built into their components: https://chakra-ui.com/docs/styled-system/style-props#grid-layout

Reply:
So sorry I added that one, I was actually trying to see if the width property was working. 
I am definitely going to take a look at the layouts!